articles/cms/wordpress/configuration.md

```diff
@@ -297,4 +297,5 @@ For more information on the Login by Auth0 WordPress Plugin, follow these links.
 * [Installation](/cms/wordpress/installation)
 * [JWT Authentication](/cms/wordpress/jwt-authentication)
 * [Troubleshoot](/cms/wordpress/troubleshoot)
+* [Extension](/cms/wordpress/extension)
```

Comment:
The link throws a 404. This should match the name of the file (so either the links will be `/cms/wordpress/extending` or the new file will be named `extension.md`.